django/thunderstore/modpacks/models/legacyprofile.py

```diff
@@ -67,3 +67,18 @@ class LegacyProfile(TimestampMixin, models.Model):
 
     def __str__(self):
         return str(self.id)
+
+
+class LegacyProfileMetaData(TimestampMixin, models.Model):
```

Comment:
In terms of naming, calling this `MetaData` is a bit redundant; just `Meta` should be fine. Not a big thing, but shorter names should be preferred where possible (also even if we want to use the longer form, it'd be `Metadata`).

I wouldn't consider it a blocker normally, but since this ends up in the db schema, it should probably be changed.

Reply:
Also this should probably be added to the cached models list given that it doesn't get a lot of writes, here: https://github.com/thunderstore-io/Thunderstore/blob/master/django/thunderstore/core/settings.py#L453